tikv-client/src/main/java/com/pingcap/tikv/TiSession.java

```diff
@@ -138,7 +138,12 @@ public Catalog getCatalog() {
     if (res == null) {
       synchronized (this) {
         if (catalog == null) {
-          catalog = new Catalog(this::createSnapshot, conf.ifShowRowId(), conf.getDBPrefix());
+          catalog =
+              new Catalog(
+                  this::createSnapshot,
+                  conf.ifShowRowId(),
+                  conf.getDBPrefix(),
+                  getConf().getLoadTables());
```

Comment:
just use conf to unify the code style, or replace all `conf` to getConf()

Reply:
change getConf to conf